src/environments/environment.ts

```diff
@@ -6,5 +6,5 @@ export const environment = {
   production: false,
   profileUrl:
     'https://landville-backend-web-api.herokuapp.com/api/v1/auth/profile/',
-  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1'
```

Comment:
Hey @MBenedicte , could we change this to use camelCase and also make it readonly?

Reply:
@CryceTruly , changing this could mean that she would have to refactor quite a lot of places in the app. Consider all the pages that are using the `api_url`. It may not be the best casing, but making changes to this might break some things

Reply:
@Tevinthuku, I understand, but we cannot compromise on conventions you know.the earlier it's done, the fewer places we will need to refactor.
We can take advantage of the web storm IDE refactoring features to change this in all occurrences just once.

Reply:
@MBenedicte kindly change as per @CryceTruly's suggestion

Reply:
As @Tevinthuku said, this would require to touch so many files that are not related to this feature. So changing the name of the api_url can be done while refactoring.